collect_app/src/main/java/org/odk/collect/android/activities/GeoTraceActivity.java

```diff
@@ -73,7 +82,7 @@
 
     private boolean beenPaused;
     private boolean modeActive;
-    private Integer traceMode = 1; // 0 manual, 1 is automatic
+    private Integer traceMode = 0; // 0 manual, 1 is automatic
```

Comment:
Previously, the initial value of 1 here had no effect: the radio group comes up with `Manual` selected (because it has `checked="true"` in the layout XML file, and the radio buttons are never updated based on `traceMode`, `traceMode` is only updated by reading the radio buttons).

Now, upon initialization, we update the radio buttons based on the restored value of `traceMode`.  To preserve the current user-visible behaviour, we need `traceMode` to be `0` so that `Manual` remains pre-selected.